common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -320,6 +359,15 @@ def test_enrollment_upgrade(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect no enrollment activated event
+        assert_event_emitted_num_times(
+            self.event_collection,
+            "edx.course.enrollment.activated",
```

Comment:
So that this test is consistent with the other two, can you move this so that it comes after `self.fake_payment_page.submit_payment()`?